Source/WebCore/platform/graphics/angle/GraphicsContextGLANGLE.cpp

```diff
@@ -1886,26 +1886,6 @@ GCGLint GraphicsContextGLANGLE::getProgrami(PlatformGLObject program, GCGLenum p
     return value;
 }
 
-String GraphicsContextGLANGLE::getUnmangledInfoLog(PlatformGLObject shaders[2], GCGLsizei count, const String& log)
```

Comment:
Comment about the PR description (not this code line) : The description mentions "getUnmangledProgramInfoLog", it should be "getUnmangledInfoLog" (no "Program").